Phish tank v2 - Fixed an issue where integrationContext was not save all data after command *url*. (#9950)
demisto/content: Packs/PhishTank/ReleaseNotes/2_0_1.md

#### Integrations
##### PhishTank v2

> - Fixed an issue where integrationContext was not save all data after command *url*.

Reviewer: ```suggestion
- Fixed an issue where the *url* command didn't cache the data.
```
